BuildTaskTests/Palaso.BuildTask.Tests.csproj

```diff
@@ -195,6 +195,7 @@
     </Compile>
     <Compile Include="ArchiveTests.cs" />
     <Compile Include="GenerateReleaseArtifactsTests.cs" />
+    <Compile Include="UnitTestTasks\NUnit3Tests.cs" />
```

Comment:
I know next to nothing about this stuff, but I'm curious. Does that backslash "\" work on both platforms here?